hpat/hiframes/hiframes_typed.py

```diff
@@ -972,7 +972,7 @@ def run_call_series_quantile_default(A):
             return self._replace_func(
                 func, (data, index, n_arg, name), pre_nodes=nodes)
 
-        if func_name in ('cov', 'corr'):
+        if func_name in ('cov'):
```

Comment:
The condition is incorrect, because `('cov')` is not tuple. BTW `('cov',)` is tuple.